docs/automation_host_prereqs.md

```diff
@@ -11,6 +12,13 @@
 
 Install the following packages on the automation host. Select the appropriate install binaries based on your automation host platform - Mac/Linux/Windows.
 
+### Configure Your Firewall
+If your system is behind a firewall, you will need to ensure the following ports are open in order to use ssh, http, and https:
+- 22, 443, 80
+
+These additional ports are required for the ocp cli (`oc`) and web console access post-install:
+- 6443
```

Comment:
Webconsole is only 443 afaik. 6443 is only for CLI. Can you please re-confirm ..